docs/storage.md

```diff
@@ -73,6 +73,16 @@ There's 2 types of storage:
     <td></td>
     <td>X</td>
   </tr>
+  <tr>
+    <td>Ceph iSCSI gateway</td>
```

Comment:
I'd put this highly experimental way of using Ceph after CephRBD, what do you think?

Reply:
Ok